automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/RegistrationSpec.scala

```diff
@@ -16,9 +16,9 @@ import org.scalatest.{BeforeAndAfter, FreeSpec, Matchers}
 /**
   * Tests for new user registration scenarios.
   */
-class RegistrationSpec extends FreeSpec with FailTestRetryable with BeforeAndAfter with Matchers with WebBrowserSpec with UserFixtures {
+class RegistrationSpec extends FreeSpec with BeforeAndAfter with Matchers with WebBrowserSpec with UserFixtures with FailTestRetryable {
```

Comment:
moving `FailTestRetryable` to after `WebBrowserSpec` because `WebBrowserSpec` override `withFixture` also. `FailTestRetryable withFixture` should be the last in call chain.